### PR TITLE
Add missing plugin descriptions

### PR DESCRIPTION
**Pull request by eko3alpha**

### CODE REVIEW

**Comment by skipjack** [comment]
@eko3alpha thanks for your contribution! I'll review this shortly once the build passes (which it should now that #1413 is in.

**Comment by skipjack** [comment]
@simon04 -

@eko3alpha added some more descriptions here and I tightened things up a bit and did some formatting. Once we fix up our markdown process and allow reference links again, this table (along with a few others) can be tightened up even further -- let me know your thoughts, I did remove some of the examples, esp filenames from the descriptions, as I don't think they were all that necessary. Short, tight descriptions here should be fine imo.

File: content/plugins/index.md [hunk 1]
```diff
@@ -5,24 +5,39 @@ contributors:
...
+[`LimitChunkCountPlugin`](/plugins/limit-chunk-count-plugin) | Set min/max limits for chunking to better control chunking
+|[`LoaderOptionsPlugin`](/plugins/loader-options-plugin) | Used for migrating from webpack 1 to 2
+|[`MinChunkSizePlugin`](/plugins/min-chunk-size-plugin)  | Keep chunk size above the specified limit
+|[`NormalModuleReplacementPlugin`](/plugins/normal-module-replacement-plugin) | Replace resource(s) that matches a regexp
+|[`NoEmitOnErrorsPlugin`](/plugins/no-emit-on-errors-plugin) | Skip the emitting phase when there are compilation errors
+|[`NormalModuleReplacementPlugin`](/plugins/normal-module-replacement-plugin) | Replace one resource with another
+|[`NpmInstallWebpackPlugin`](/plugins/npm-install-webpack-plugin) | Auto-install missing dependencies during development
+|[`ProvidePlugin`](/plugins/provide-plugin)              | Use modules without having to use import/require
+|[`SourceMapDevToolPlugin`](/plugins/source-map-dev-tool-plugin) | Enables a more fine grained control of source maps
+|[`UglifyjsWebpackPlugin`](/plugins/uglifyjs-webpack-plugin)     | enables control of the version of UglifyJS in your project
```
Comment (simon04) [inline content/plugins/index.md]: enables→Enables (for consistency)

File: content/plugins/index.md [hunk 2]
```diff
@@ -5,24 +5,39 @@ contributors:
...
+|[`ZopfliWebpackPlugin`](/plugins/zopfli-webpack-plugin) | Node-Zopfli plugin for webpack
```
Comment (simon04) [inline content/plugins/index.md]: The GitHub project description is more informative: "Prepare compressed versions of assets with node-zopfli"

**Pull request merged**